data/en/runasync.json

```diff
@@ -0,0 +1,46 @@
+{
+    "name":"runAsync",
+    "type":"function",
+    "syntax":"runAsync(function() {})",
+    "script":"runAsync(function() {});",
+    "returns":"Future",
```

Comment:
I do not see that they added this as a new return type. This should probably be `any`.

Reply:
They say, it is called `Future` which is something like a `Promise` in JavaScript. Should we really use something unspecific like `any`?

Reply:
@shaedrich To be clear, `Future` in CFML is nothing like a `Promise` in JavaScript (its a subtle difference, but is different enough not to confuse the two). It is closer to a `CompletableFuture` in Java. 

That said, and in this case, I think `any` is probably appropriate here given it returns any value... it just happens to return it at a future point in the code (which could perhaps be spelled out in the description, if it isn't already)